modules/service/src/main/scala/higherkindness/mu/rpc/channel/ManagedChannelInterpreter.scala

```diff
@@ -59,8 +57,8 @@ class ManagedChannelInterpreter[F[_]](
     } yield built
   }
 
-  def unsafeBuild[T <: ManagedChannelBuilder[T]](implicit E: Effect[F]): ManagedChannel =
-    E.toIO(build).unsafeRunSync()
+  def unsafeBuild[T <: ManagedChannelBuilder[T]](disp: Dispatcher[F]): ManagedChannel =
```

Comment:
This is probably the most annoying code change that *might* be user-facing.

It seems like in the codebase here, the main callers of this are macro-generated code, which I was able to adjust to construct a Dispatcher